docs/source/introduction/ext_cfg_util.rst

```diff
@@ -0,0 +1,73 @@
+The purpose of the VISA external configuration utility
+==============================================================================
+
+Managing system VISA resources -
+------------------------------------------------------------------------------
+The three main VISA distributions all come with a utility to manage VISA
+resource configurations.  This utility is refered to as the external
+configuration utility.  The VISA external configuration utility is used to
+manage instrument attribute used to configure buses, instruments, and aliases.
+This configuration in turn can be used by pyVISA to manage instrument
+configurations in a more user friendly manner than is otherwise afforded.
+
+.. note::
+
+    Not all VISA implementations handle settings configured in the external
+    utility the same.  For instance, in NI-VISA, if a serial port is configured
+    for baudrate = 115000, the default value will still be configured as 9600.
+    To pass the VISA configured value to a new instrument instance, access mode
+    must be set to constants.VI_LOAD_CONFIG (see below), otherwise, the default
+    value of 9600 will be passed to the instrument instance.  R&S-VISA doesn't
+    work this way.  (I was not able to load Keysight VISA to test it's
+    functionality at the time of this writing.)
```

Comment:
Since I won't have the bandwidth to test, this will remain, so maybe rephrase in a less personal manner.